### DISCUSSION TITLE
Problema na autenticação

### DISCUSSION THREAD

**Discussion by helloimpeache** [andreroggeri/pynubank#421] [Q&A]
The request made failed with HTTP status code 401
  File "C:\Users\TESTE-PY.py", line 5, in <module>
    print(nu.get_account_balance())
          ^^^^^^^^^^^^^^^^^^^^^^^^
pynubank.exception.NuRequestException: The request made failed with HTTP status code 401

Meu código está dando unauthorized, sendo que todas minhas credenciais estão corretas. Uso essa lib há um tempo e só agora foi dar problema.

**Comment by andreroggeri** [comment]
Duplicado com a #419
